libhoth_usb.h

```diff
@@ -44,31 +29,6 @@ struct libhoth_usb_device_init_options {
 };
 
 int libhoth_usb_open(const struct libhoth_usb_device_init_options* options,
```

Comment:
Make it clear in the docs that the options pointer only needs to live as long as this function call.

Reply:
Clarified